default.nix

```diff
@@ -0,0 +1,25 @@
+# copied expressions from https://nixos.wiki/wiki/Rust
```

Comment:
You! Sneaky!

Reply:
:grimacing: actually didn't mean to add this :sweat_smile: But while we're here... Would you prefer it didn't live in the repo or is it ok to merge? :eyes:

Reply:
Nix los hier.